### PR TITLE
[Instrument Builder] Disable 'Load Instrument' button when text box is empty

### PR DESCRIPTION
**Pull request by victoriafoing**
This pull request disables the 'load instrument' button when the text box is empty.

This is a small UI/UX improvement addressing the following ticket: https://redmine.cbrain.mcgill.ca/issues/12002

disabled

enabled

### CODE REVIEW

File: modules/instrument_builder/jsx/react.instrument_builder.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ var LoadPane = React.createClass({
       // This is used to alert the user if the file was
       // loaded successfully or there was an error with
       // the loading.
+      disabled: true,
```
Comment (driusan) [inline modules/instrument_builder/jsx/react.instrument_builder.js]: The comment above this is now wrong, the comment is for the alert, not the disabled attribute..

Comment (victoriafoing) [reply modules/instrument_builder/jsx/react.instrument_builder.js]: Thanks. I moved it above the comments in my latest commit.

**Comment by victoriafoing** [comment]
Let me know if you think it is unnecessary to set disabled as true in the chooseFile function since it is already set as true in the getInitialState function.
e.g.
this.setState({
      file: value,
      **disabled: true,**
      alert: ''
    });

**Comment by colezweber** [comment]
Looks good on Firefox and Chrome

**Pull request merged**